tests/regression/test_bcs.py

```diff
@@ -113,6 +113,16 @@ def test_homogenize_doesnt_overwrite_function(a, u, V, f):
     solve(a == 0, u, bcs=[bc])
     assert abs(u.vector().array()).max() == 0.0
 
+def test_homogenize(V):
+    bc = [DirichletBC(V, 10, 1), DirichletBC(V, 20, 2)]
+
+    homogeneous_bc = bcs.homogenize(bc)
```

Comment:
Maybe you want to add 'homogenize' to __all__ in bcs.py, so that you don't need to prefix it with the module name.